spec/models/resource_group_spec.rb

```diff
@@ -23,8 +23,39 @@
   end
 
   context "relationships" do
+    before do
+      @vm       = FactoryGirl.create(:vm_amazon, :template => false, :resource_group => resource_group)
+      @template = FactoryGirl.create(:vm_amazon, :template => true, :resource_group => resource_group)
+    end
+
+    it "has many vms_or_templates" do
+      expect(resource_group).to respond_to(:vm_or_templates)
```

Comment:
I know people disagree with me on tests like this, but I like them because they're good for sanity checking. At worst, it's harmless.